Song_sort_Title.php

```diff
@@ -1,48 +1,81 @@
+<?php
+		// Deliver to login page if access to this page with out login
+		session_start();
+		if(!isset($_SESSION['login_user'])){
+				header("location:login.php");
+				}
+		else{
+			$User = $_SESSION['login_user'];
+		}
+?>
+
 <!DOCTYPE html>
 <html lang = "en">
 	<head>
-		<title>Layout1Musicplay</title>
+		<title>Sort By Title</title>
```

Comment:
21/9: Change the title for the page for it to be right. So I will not get confuse when I come back to look at this (also for other people come to look at my code)